src/components/NewEvent.js

```diff
@@ -26,11 +27,18 @@ const NewEvent = () => {
             guests: formValues.guests.trim(),
             food: formValues.food.trim(),
         }
+        setPotluck(potluck.concat(newPotluck))
+        setFromValues({date: '', location: '', guests: '', food: ''})
```

Comment:
This says setF**r**omValues instead of setFo**r**mValues

Reply:
it's the same as your state so it should be okay